test/inductor/test_memory.py

```diff
@@ -306,6 +306,57 @@ def f(a, b, c):
         expected_bound = a.size(0) * c.size(1) * a.dtype.itemsize * 2
         self.assertLess(peak_mem, expected_bound)
 
+    def test_fusion_acc_large_reads(self):
```

Comment:
Is this test use lots of GPU memory? If that's the case, you could try to add `@serialTest()` decorator to avoid running it in parallel with other tests, which might cause OOM